devicon.json

```
@@ -1,4 +1,32 @@
 [
+    {
+        "name": "aerospike",
+        "altnames": [
+            "aerospike"
+        ],
+        "tags": [
+            "data",
+            "database",
+            "nosql"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original",
+                "original-wordmark"
+            ]
+        },
+        "color": "#c21417",
+        "aliases": [
+            {
+               "base": "original",
+               "alias": "plain"
+            }
```

Since the `original-wordmark` version is mono-colored and has a single path, it can also be used as a plain version. To make sure that the plain version "copies" the original version you can add an alias between the two like this:
```suggestion
            },
            {
               "base": "original-wordmark",
               "alias": "plain-wordmark"
            }
```